src/System.Text.Encodings.Web/src/Configurations.props

```diff
@@ -1,7 +1,7 @@
 ﻿<Project DefaultTargets="Build">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
+      netcoreapp3.0;
```

Comment:
Some of the references that S.T.E.W depends on don't have a config for netcoreapp3.0.
`System.Diagnostics.Debug`
`System.Memory`
`System.Runtime`
etc.

Should we also add the versioned TFM configs to all those (along side the version-less ones)?

https://github.com/dotnet/corefx/blob/bd18c12df9e5dd365a8b708cc2e7e0488b8177c1/src/System.Diagnostics.Debug/src/Configurations.props#L4-L5

https://github.com/dotnet/corefx/blob/bd18c12df9e5dd365a8b708cc2e7e0488b8177c1/src/System.Memory/src/Configurations.props#L4-L5

That's causing the CI failures:
```text
##[error].dotnet\sdk\3.0.100\Microsoft.Common.CurrentVersion.targets(2106,5): error MSB3245: Could not resolve this reference. Could not locate the assembly "System.Runtime". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors.

```

Reply:
`build -allconfigurations` succeeds but targeting individual frameworks (particularly the default - netcoreapp) fails.

Reply:
This library is inbox as well so you will need a version-less build configuration but you will need to exclude it from the packages configuration so that it targets the latest.

So you do something like:

```xml
<PackageConfigurations>
  netcoreapp3.0;
  netstandard2.1;
  netstandard;
</PackageConfigurations>
<BuildConfigurations>
  $(PackageConfigurations);
  netcoreapp;
</BuildConfigurations>
```

Reply:
Awesome, that worked! I wasn't updating the `Configurations` property in the csproj locally to include `netcoreapp` which is why I was seeing the failure earlier.